src/utils/add.trust.ts

```diff
@@ -31,6 +31,7 @@ type TrustInvolvedPageProperties = {
   formData?: PageModel.AddTrust,
   errors?: FormattedValidationErrors,
   url: string,
+  isFeatureFlagCeaseTrustsEnabled: boolean
```

Comment:
Not sure why this is in the data structure, but question might be answered by response to [this](https://github.com/companieshouse/overseas-entities-web/pull/1345/files#r1523493973) comment.

Reply:
it's because i'm passing the feature flag to the template and that is the object type that is passed so needs to be added there or you get errors

Reply:
Ok. All clear.